pkg/transport/network/tcp/server.go

```diff
@@ -0,0 +1,131 @@
+package tcp
+
+import (
+	"net"
+	"time"
+
+	"github.com/HyperloopUPV-H8/h9-backend/pkg/abstraction"
+)
+
+// Address is an alias for string encoded network addresses (e.g. "127.0.0.1:4040")
+type Address = string
+
+// serverTargets are the addresses are expected to connect and their respective target name
+type serverTargets = map[Address]abstraction.TransportTarget
+
+// Assertion to check the TCPServer is a TCPSource
+var _ Source = &Server{}
```

Comment:
What is the purpose of this? I mean, you don't use it for anything.

Reply:
Read the comment

Reply:
But if it doesn't follow it, what is the difference in the execution?